README.md

```diff
@@ -536,7 +534,7 @@ spec:
 Run following command to apply Trigger.
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_trigger.yaml
+$ oc create -f https://github.com/openshift/pipelines-tutorial/blob/pipelines-1.4/03_triggers/03_trigger.yaml
```

Comment:
change to `https://raw.githubusercontent.com/openshift/pipelines-tutorial/pipelines-1.4/03_triggers/03_trigger.yaml`

Reply:
Updated